onnxruntime/core/codegen/common/dispatcher.h

```diff
@@ -68,6 +69,7 @@ class DispatcherBase {
   std::string name_;
   std::unordered_map<std::string, CONTENT_TYPE> contents_;
   ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(DispatcherBase);
+  ~DispatcherBase() = default;
 };
```

Comment:
Should this be virtual?

Reply:
This is protected dtor so OK to be not virtual. [http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-dtor-virtual](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-dtor-virtual)

---
In reply to: [313174357](https://github.com/microsoft/onnxruntime/pull/881#discussion_r313174357) [](ancestors = 313174357)